server/utils/zip.js

```diff
@@ -4,12 +4,26 @@ import JSZip from 'jszip';
 import tmp from 'tmp';
 import unescape from '../../shared/utils/unescape';
 import { Collection, Document } from '../models';
+import { getImageByKey } from './s3';
+
+const ENABLE_PRIVATE_CONTENT = process.env.ENABLE_PRIVATE_CONTENT === 'true';
+const s3KeyRegex = /!\[.*\]\(\/api\/images\.info\?key=(?<key>.*)\)/gi;
+const imgageApiRegex = /(?<=!\[.*\]\()(\/api\/images\.info\?key=)/gi;
```

Comment:
```suggestion
const imageApiRegex = /(?<=!\[.*\]\()(\/api\/images\.info\?key=)/gi;
```

Although, I wonder if these two regex could be combined :thinking:

Reply:
I'll have a think, I'm sure it is possible to merge. I had 1 regex pulling out just the s3 key part as I need that to get the object from s3, and the second one was specifically to pull out the `/api/images.info?key=` substring for replacement. The complexity with the latter is that I needed to make sure to not accidentally match on someone writing `/api/images.info?key=` in the document text (who would, I know!), hence the lookbehind check to ensure I'm working on the image urls only.

Could be dropped altogether pending our below discussion on an assets table.